packages/angular/projects/clr-angular/src/data/datagrid/datagrid-row.ts

```diff
@@ -133,12 +135,9 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
   }
 
   // By default every item is selectable
-  @Input('clrDgSelectable')
-  public set clrDgSelectable(value: boolean) {
-    this.selection.lockItem(this.item, value === false);
-  }
+  @Input('clrDgSelectable') clrDgSelectable: boolean;
```

Comment:
If every item is selectable by default, should this be set to `true` by default?

Reply:
The reason for not adding a default value is that - we only care if it's `false` and everything else is `true` for us. But I could change it it's an easy one ;)

Reply:
Yeah, I guess logically it makes no difference. =) It was a bit harder to parse that mentally upon initial glance. In some parts of our code I think we rely on falsy value as equivalent of setting it to false (which is not true in this case).

Maybe we don't need to change, and just edit the comment?
```
// By default every item is selectable; it becomes not selectable only if it's explicitly set to false
```